ci/github-actions.md

```diff
@@ -0,0 +1,125 @@
+# GitHub Actions
+
+This guide provides documentation for setting up continuous integration with [GitHub Actions](https://help.github.com/en/actions) for different types of projects.
+
+## Contents
+
+- [Running containerized tests](#running-containerized-tests)
+- [Deploying a Python package](#deploying-a-python-package)
+- [Resources](#resources)
+
+## Running containerized tests
+
+To initialize a workflow for running containerized tests, start by creating a new feature branch in your repo. Then, create a directory called `.github/workflows/` at the root of your repo. GitHub Actions will read any workflows from this directory and run them automatically.
+
+Next, define a file called `main.yml` in the workflows directory, and paste in the following configuration:
+
+```yaml
+name: CI
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Build containers and run tests
+      run: docker-compose -f docker-compose.yml -f tests/docker-compose.yml run --rm app
+```
+
+Commit this file to your feature branch and open up a pull request. You should be able to confirm that your workflow runs the tests for your pull request.
+
+### Examples
+
+This workflow is in use in the following projects:
+
+- [Minnesota Election Archive](https://github.com/datamade/mn-election-archive) (visible to DataMade developers only)
+
+## Deploying a Python package
+
+GitHub Actions can run arbitrary code, so we can use it to deploy Python packages to PyPI in addition to testing them.
```

Comment:
Again, I think it would be useful to have a sentence here about what the following code does. So something like: 

> These two Actions will publish any branch merged into `master` to [test PyPI](https://test.pypi.org/) and publish tagged commits to [PyPI](https://pypi.org/). This structure parallels our practice of keeping a staging site consistent with `master` and pushing tagged commits to production.

Reply:
This is great, thanks! I added a slightly altered version of this copy below.